lib/mktorrent.rb

```diff
@@ -19,6 +20,7 @@ def initialize(tracker)
     @files = []
     @filehashes = []
     @size = 0
+    @tracker_list = Array.[](Array.[](@tracker))
```

Comment:
"announce-list: (optional) this is an extension to the official specification, offering backwards-compatibility. (list of lists of strings)."